src/odemis/acq/align/__init__.py

```diff
@@ -22,102 +22,8 @@
 
 from __future__ import division
 
-import logging
-from odemis.model import MD_PIXEL_SIZE
 from .autofocus import AutoFocus, AutoFocusSpectrometer, Sparc2AutoFocus
 from .light import turnOnLight
 from .find_overlay import FindOverlay
 from .spot import AlignSpot, FindSpot
 from .fastem import align
```

Comment:
Wouldn't the circular dependency also be solved by not importing this function? (ie, deleting this line).

Typically we have these imports in order to hide the fact that the functions are placed in submodules. However, typically these "hidden" submodules have names starting with a `_`. Anyway, the `align()` function has a too generic name to just be loaded as `acq.align.align()`.

Reply:
I was sure I had tried that, but you are correct it works! And I agree that the `align` function has a too generic name to be loaded like that.

I will move back FindEbeamCenter and remove the line `from .fastem import align`